src/cpp/include/openvino/genai/image_generation/image_generation_perf_metrics.hpp

```diff
@@ -0,0 +1,52 @@
+// Copyright (C) 2023-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <vector>
+#include <chrono>
+#include <map>
+#include <string>
+
+#include "openvino/genai/visibility.hpp"
+#include "openvino/genai/perf_metrics.hpp"
+
+namespace ov::genai {
+
+struct OPENVINO_GENAI_EXPORTS RawImageGenerationPerfMetrics {
+    std::vector<MicroSeconds> unet_inference_durations; // unet inference durations for each step
+    std::vector<MicroSeconds> transformer_inference_durations; // transformer inference durations for each step
+    std::vector<MicroSeconds> iteration_durations;  //  durations of each step
+};
+
+struct OPENVINO_GENAI_EXPORTS ImageGenerationPerfMetrics {
+    float load_time; // model load time (includes reshape & read_model time), ms
+    float generate_duration; // duration of method generate(...), ms
+
+    MeanStdPair iteration_duration; // Mean-Std time of one generation iteration, ms
+    std::map<std::string, float> encoder_inference_duration; // inference durations for each encoder, ms
+    MeanStdPair unet_inference_duration; // inference duration for unet model, should be filled with zeros if we don't have unet, ms
+    MeanStdPair transformer_inference_duration; // inference duration for transformer model, should be filled with zeros if we don't have transformer, ms
+    float vae_encoder_inference_duration; // inference duration of vae_encoder model, should be filled with zeros if we don't use it, ms
+    float vae_decoder_inference_duration; // inference duration of vae_decoder model, ms
+
+    bool m_evaluated = false;
```

Comment:
maybe need to hide this field into a `private` section? Looks like it's not useful for users.

Reply:
That means we also need to provide functions for setting these parameters, right?

Reply:
hm, forgot about it.

Ok, let's keep fields as is